crates/solvers/src/domain/naive.rs

```diff
@@ -0,0 +1,86 @@
+//! "Naive" solver implementation.
+//!
+//! The naive solver is a solver that collects all orders over a single token
+//! pair, and matches the excess over a Uniswap V2 pool.
```

Comment:
I don't understand what you mean by "matches the excess over a Uniswap V2 pool".

Reply:
What ever can't be matched peer-to-peer. Will try and clarify.

Reply:
12e2d3205a6b8ac29ac655129fd3c4d687e74ef5